rules/integrations/gcp/credential_access_gcp_kubernetes_secrets_enumeration.toml

```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic","Austin Songer"]
+description = """
+Identifies the unauthorized enumeration of kubernetes secrets by a user.
+"""
+from = "now-20m"
```

Comment:
We need to specify in which instances this operation would be used by administrators in false positives.